src/brokers/commerzbank.js

```diff
@@ -46,12 +49,23 @@ const findTimeSell = textArr =>
 
 const findDateDividend = (textArr, foreignDividend = false) => {
   if (foreignDividend) {
-    return textArr[textArr.findIndex(t => t.includes('Information')) - 3];
+    if (textArr.indexOf('Information') > 0) {
+	  return textArr[textArr.findIndex(t => t.includes('Information')) - 3];
+	}
+  }
+  if (textArr.indexOf('Valuta') > 0) {
+	if (textArr[textArr.findIndex(t => t.includes('Valuta')) + 1].split(".").length > 1) {
+      return textArr[textArr.findIndex(t => t.includes('Valuta')) + 1]
+	} 
+  }
+  if (textArr.indexOf('per') > 0) {
+	if (textArr[textArr.findIndex(t => t.includes('per')) + 1].split(".").length > 1) {
```

Comment:
Did you run the eslint pre-commit hock at commit? The hock will fix this automatically

```suggestion
	if (textArr[textArr.findIndex(t => t.includes('per')) + 1].split('.').length > 1) {
```

Reply:
Fixed